docs/build/configure/webhooks.md

```diff
@@ -11,35 +11,91 @@ Webhooks allow you to trigger actions when certain types of data are sent from y
 For example, you can configure a webhook to send you a notification when your robot's sensor collects a new reading.
 Viam provides two webhook types depending on the event you want to trigger on:
 
-- `"part_data_ingested"`: trigger when a sensor on the machine part collects a new reading
-- `"part_online"`: trigger when the machine part is online
+- Data has been synced to the cloud: trigger when data from the machine is synced
+- Part is online: trigger continuously at specified interval while the {{< glossary_tooltip term_id="part" text="machine part" >}} is online
+- Part is offline: trigger continuously at specified interval while the machine part is offline
 
 To configure a webhook:
 
-{{< tabs name="Types of Webhooks" >}}
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
 
 1. Go to the **CONFIGURE** tab of your machine on the [Viam app](https://app.viam.com).
-   Select **JSON** mode.
+   Click the **+** (Create) button in the left side menu and select **Webhook**.
+
+   {{<imgproc src="/build/configure/webhook-create.png" resize="x400" declaredimensions=true alt="The Create menu with Webhook at the bottom of the list of options." >}}
+
+2. Name the webhook and click **Create**.
+
+3. Select the type of event to trigger on from the **Type** dropdown.
+
+4. Follow the instructions depending on the type of webhook you want to implement:
+
+{{< tabs name="Types of Webhooks" >}}
+{{% tab name="Data synced to cloud" %}}
 
-2. Follow the instructions depending on the type of webhook you want to implement:
+5. Select the types of data you want to trigger on from the dropdown.
 
-{{% tab name="part_data_ingested" %}}
+{{% alert title="Note" color="note" %}}
+Be sure to configure [data capture](/data/capture/) and [cloud sync](/data/cloud-sync/) for the relevant components.
+For example, if you want to trigger a webhook on temperature readings, configure data capture and sync on your temperature sensor.
+Be aware that the component must return the type of data you configured in the webhook's **Data Types**.
```

Comment:
```suggestion
Be aware that the component must return the type of data you configure in the webhook's **Data Types**.
```
I think?
Seems slightly unclear how it matches to components-- will for example "Binary (image)" data trigger from any camera? Maybe could add more info here

Reply:
Okay yeah I think configured or configure works--will change! Yes it will trigger whenever data of that type is synced to the cloud; I don't think it has anything to do with which component it's coming from?